source/chapters/1/intro.md

```diff
@@ -148,13 +170,13 @@ That's it. You now have added bootstrap in your application. And it's ready for
 There are a lot of cool and useful packages out there that we'll explore during these tutorials.
 
 ## Importing from Atmosphere
-You also can use the modular approach with Meteor packages:
+You also can use the modular approach with Meteor packages!
 
-As a sample, we are using SimpleSchema, a package that allows us to easily validate objects.
+As a sample, we will use SimpleSchema, a package that allows us to easily validate objects.
```

Comment:
This sample, is not really good anymore, SimpleSchema fully migrated to NPM, is  independent right now.
As a sample, you can say about Grapher
https://atmospherejs.com/cultofcoders/grapher

Reply:
Done. Waiting for feedback regarding the code sample i added.